contrib/babelfishpg_tsql/src/backend_parser/gram-tsql-rule.y

```diff
@@ -33,11 +33,11 @@ tsql_CreateLoginStmt:
 											@1)); /* Must be first */
 					n->options = lappend(n->options,
 										 makeDefElem("createdb",
-													 (Node *)makeBoolean(true),
+													 (Node *)makeBoolean(false),
 													 @1));
 					n->options = lappend(n->options,
 										 makeDefElem("createrole",
-													 (Node *)makeBoolean(true),
+													 (Node *)makeBoolean(false),
```

Comment:
Does this "createrole" permission control create/drop/alter role?

Reply:
Yes, now "createrole" permission will be granted to a login only via the TDS port using the below statement.
`alter server role sysadmin add member login1`